test1-dev.js

```diff
@@ -3,3 +3,9 @@ var foo = '' ;
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var ananas = '' ;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-test-dev/pullRequest?prid=27791)
```suggestion
var ananas = "" ;
```